tests/data/okta/userfactors.py

```diff
@@ -0,0 +1,16 @@
+from datetime import datetime
+
+from okta.models.factor.Factor import Factor
+
+
+def create_test_factor():
+    factor = Factor()
+
+    factor.id = "jkljsdf"
```

Comment:
Please use real values for these fields. When tests fail it should be easy to understand what the issue is. Failures like `None != "jkljsdf"` will not be helpful.